### ISSUE TITLE
Table row resizing

### ISSUE THREAD

**Issue by oleghind** [ckeditor/ckeditor-dev#737]
is there any way to allow table row resizing (row's height)? Seems like your competitors have this feature, this is something that prevents us from purchasing the enterprise license, let me know if you have any plans on this.

**Comment by f1ames** [comment]
There is a [`tabletools` plugin](https://ckeditor.com/addon/tabletools) which adds advanced options to table context menu. You can use `Cell Properties` dialog
![](https://ckeditor.com/sites/default/files/styles/large/public/tabletools/table_04.png?itok=fYl1igke)
 to set cells height so the whole table row gets resized (the table row height is the height of the highest cell in this row).

You can test it on the [full preset SDK sample](http://sdk.ckeditor.com/samples/fullpreset.html).

---

Closing this issue as invalid as it is not a bug report nor a feature request.

**Comment by f1ames** [comment]
@oleghind just to clarify, you may be thinking about row resizing using mouse (dragging borders) just like column resizing [works here](http://sdk.ckeditor.com/samples/table.html). If this is what you were thinking of, we may treat this issue as a feature request.
However, such feature is not present in CKEditor and is not a high priority for now, so it is not planned for the nearest future.

**Comment by Stalq** [comment]
@oleghind in the light of what @f1ames wrote we'll be delighted to continue discussion with you and your team via our sales channel. If that feature is high priority for you then we might cooperate. Please reach out to us either via sales@cksource.com or via contact form on https://ckeditor.com/contact. We look forward to your message.

**Comment by oleghind** [comment]
Yes I am talking about resizing by dragging. Also if you check google docs, you'll see that you can actually select a piece of border and change it's thickness, color or pattern. (that's not a dealbreaker however, but a very neat feature to have). But the row resizing is pretty essential. We've just started testing ckeditor solution and will contact sales for sure if there is not too many features missing. Thanks!

**Comment by f1ames** [comment]
@oleghind, I changed this issue to feature request and updated the description so it is more clear for other users. I encourage you to contact sales to discuss all possible ways how it could be handled.
